lib/Bio/EnsEMBL/DataCheck/Checks/CheckNonMinimisedGATs.pm

```diff
@@ -0,0 +1,79 @@
+=head1 LICENSE
+
+Copyright [2018-2021] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the 'License');
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an 'AS IS' BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package Bio::EnsEMBL::DataCheck::Checks::CheckNonMinimisedGATs;
+
+use warnings;
+use strict;
+
+use Moose;
+use Test::More;
+use Bio::EnsEMBL::DataCheck::Test::DataCheck;
+
+extends 'Bio::EnsEMBL::DataCheck::DbCheck';
+
+use constant {
+  NAME           => 'CheckNonMinimisedGATs',
+  DESCRIPTION    => 'Check that all genomic align trees are binary, with the exception of unrooted EPO-extended trees which may have 3 sequences',
+  GROUPS         => ['compara', 'compara_genome_alignments'],
+  DATACHECK_TYPE => 'critical',
+  DB_TYPES       => ['compara'],
+  TABLES         => ['genomic_align_tree', 'method_link_species_set']
+};
+
+sub tests {
+  my ($self) = @_;
+
+  my $desc_1 = "All genomic_align_tree trees are correct";
```

Comment:
Not sure if this description is the best, but I think it may provide more information for people from other groups/newcomers:
```suggestion
  my $desc_1 = "All genomic_align_tree trees are binary (or ternary if EPO-extended and 4-node tree)";
```